src/components/v5/shared/HoverWidthWrapper/HoverWidthWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import { HoverWidthWrapperProps } from './types';
+
+const HoverWidthWrapper: FC<HoverWidthWrapperProps> = ({
+  hoverClassName,
+  hoverElement: HoverElement = 'div',
+  children,
+}) => (
+  <>
+    {children}
+    <HoverElement
+      className={clsx(
+        'font-medium text-transparent h-[.0625rem] overflow-hidden invisible',
```

Comment:
I wouldn't add `font-medium` here as this style is specific to a certain use case and should be passed trough `hoverClassName`.